docs/source/managed-federation/deployment.mdx

```diff
@@ -233,7 +233,7 @@ If the Apollo Studio API experiences an outage, changes to your configuration wi
 
 ## The `service:push` lifecycle
 
-Whenever you call `apollo service:push` for an implementing service, it both updates that service's registered schema and updates the gateway's managed configuration. Because multiple services might be updated simultaneously, it's possible for changes to cause composition errors even if 
```

Comment:
I'm imagining the original author (@StephenBarlow) had something in mind here.

Even if..."`apollo service:check` was successful", maybe? Stephen, would you mind chiming in on this please?

Reply:
Ha whoops! Not sure what happened here, but yes, that looks like the correct end of this sentence. @DonnyVerduijn could you update this PR to reflect @trevor-scheer 's suggestion?

Reply:
The paragraph below also rephrases the same topic, so i merged them together now. I'm not sure about the last sentence. It might need its own paragraph and refer to the paragraph above?

Reply:
Thanks @DonnyVerduijn, this looks good! I'll revisit this article in the near future to see about some more general structural improvements.

Reply:
Thanks for merging @StephenBarlow. I agree there's still room for some polishment in the federation docs. Definitely a nice challenge ahead!